### ISSUE TITLE
[Gestion des droits] configuration dans le cadre de plusieurs organismes

### ISSUE THREAD

**Issue by JeromeMaruejouls** [PnX-SI/GeoNature#1854]
Bonjour,
Voici mon problème : 
 Je gère un géonature pour plusieurs organismes : 
- chaque utilisateur voit toutes les données mais ne télécharge que les données de son organisme

Cela se configure facilement avec le CRUVED de l'outils synthèse. (Utilisation de visible par tous et exportable pour l'organisme)

Mais j'aimerai aussi avoir des JDD dans la base, visibles par seulement un organisme, un utilisateur, voire de personne.

Comment configurer l'ensemble, et est ce possible ?

**Comment by TheoLechemia** [comment]
Bonjour,
Non, il n'existe pas actuellement de possibilité pour avoir une règle spéciale pour un ou des JDD.

**Comment by DonovanMaillard** [comment]
Sauf erreur, le cruved permet pourtant de donner accès uniquement aux métadonnées de son propre organisme. Et de mémoire ça fonctionnait sur mon instance multi-organismes :/ 

Les seules perméabilités que j'avais entre plusieurs organismes sont les listes d'acteurs, les listes d'espèces dans l'autocomplétion de la synthèse et les listes d'observateurs, donc rien concernant l'accès aux mtd

**Comment by TheoLechemia** [comment]
Je crois que ce que @JeromeMaruejouls voulait faire, c'est que toute le monde voir tout (R=3), mais que certains JDD ne soient visible que par un organisme et pas par d'autre.
Dis moi si j'ai mal compris

**Comment by JeromeMaruejouls** [comment]
Oui, gérer plusieurs organismes fonctionne correctement. Mais j'aimerai notamment pouvoir exclure un JDD du R=3, pour que les données restent dans la base, mais ne soient plus visibles (cas de données mises à disposition temporairement, dans le cadre d'une étude spécifique).
